src/pl/plpython/plpy_main.c

```diff
@@ -137,6 +157,18 @@ _PG_init(void)
 	prev_cancel_pending_hook = cancel_pending_hook;
 	cancel_pending_hook = PLy_handle_cancel_interrupt;
 
+#if PY_MAJOR_VERSION >= 3
+	DefineCustomStringVariable("plpython3.python_path",
```

Comment:
What does `u` mean in the `plpython3u`? I am thinking about shall we align the GUC name with the pl name `plpython3u` like `plpython3u.python_path` here?

Reply:
I think we should follow the `plperl` and 'plpgsql' logic which `guc` name has not `u`